contentcuration/contentcuration/static/js/edit_channel/models.js

```diff
@@ -178,10 +180,10 @@ var ContentNodeCollection = BaseCollection.extend({
 		});
         return promise;
 	},
+	comparator:function(node){
+		return node.get("sort_order");
+	},
     sort_by_order:function(){
```

Comment:
You could overwrite the base Backbone sort here, so that this attribute gets set automatically on sorting.
```
sort: function() {
Backbone.Model.prototype.sort.call(this);
this.highest_sort_order = (this.length > 0)? this.at(this.length - 1).get("sort_order") : 1;
},
```
and then you don't have to call this function at all, you'll just keep track of the highest_sort_order whenever a resort happens.